### PR TITLE
#94 add friends resource

### PR DESCRIPTION
**Pull request by denys-medynskyi**

### CODE REVIEW

File: app/controllers/admin/friends_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      @friend ||= Friend.find(params[:id])
+    end
+
+    def friends
+      @friends ||= Friend.order(:name)
+    end
+
+    def friend_params
+      params.require(:friend).permit(
+          :name, :group_id, :logo, :link, :description
```
Comment (kushniryb) [inline app/controllers/admin/friends_controller.rb]: Use 2 spaces for identation :)

File: app/controllers/admin/friends_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+module Admin
+  class FriendsController < BaseController
+    helper_method :friend, :friends, :venues
```
Comment (kushniryb) [inline app/controllers/admin/friends_controller.rb]: Am I missing something here? There's no `venues` method

**Comment by kushniryb** [comment]
LGTM overall :+1:

**Review by kushniryb — APPROVED** [review]

File: app/controllers/admin/friends_controller.rb (hunk 1 above)
Comment (sveredyuk) [reply app/controllers/admin/friends_controller.rb]: rubymine always add 4 spaces for hashes..  wtf rubymine ?

Comment (kushniryb) [reply app/controllers/admin/friends_controller.rb]: Vim for the win ^_^

**Review by sveredyuk — COMMENTED** [review]
And where at main app we will show friends ?

**Review by sveredyuk — APPROVED** [review]

File: app/controllers/admin/friends_controller.rb (hunk 2 above)
Comment (denys-medynskyi) [reply app/controllers/admin/friends_controller.rb]: copy-pasting rules

File: app/controllers/admin/friends_controller.rb (hunk 1 above)
Comment (denys-medynskyi) [reply app/controllers/admin/friends_controller.rb]: thanks for comments

**Comment by denys-medynskyi** [comment]
@sveredyuk I wasn't sure where/how to show friends. Should it be list of friends ( http://www.rug.si/sponsors/ ) or friends will be visible on home page as sponsors ( http://vilniusrb.lt/ )

**Comment by sveredyuk** [comment]
@denysmedynskyi let leave this decision to our designer... )

**Pull request merged**